addons/csw/functions/fnc_canGetIn.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+ * Author: TCVM
+ * Checks if the player can get in the weapon
+ *
+ * Arguments:
+ * 0: Static Weapon <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [weapon] call ace_csw_fnc_canGetIn
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_staticWeapon"];
+
+alive _staticWeapon && (((crew _staticWeapon) isEqualTo [] || {!(alive (gunner _staticWeapon))}))
```

Comment:
check if locked

Reply:
Why check both crew and gunner?

Reply:
According to the biki `crew` returns both alive and dead members

Reply:
So the alive gunner check should be enough? :p